pkg/dyninst/actuator/errors.go

```diff
@@ -26,3 +26,17 @@ func (e *NoSuccessfulProbesError) Error() string {
 		"has no successful probes, contains %d issues", len(e.Issues),
 	)
 }
+
+// Format implements the fmt.Formatter interface.
+func (e *NoSuccessfulProbesError) Format(f fmt.State, _ rune) {
+	fmt.Fprintf(f, "ProbeIssues: ")
+	for i := 0; i < min(len(e.Issues), 3); i++ {
+		fmt.Fprintf(f, "%s: %s", e.Issues[i].Kind.String(), e.Issues[i].Message)
+		if i < len(e.Issues)-1 {
+			fmt.Fprintf(f, ", ")
+		}
+	}
+	if len(e.Issues) > 3 {
+		fmt.Fprintf(f, "...")
+	}
+}
```

Comment:
I think we should say something in this string that there were no successful probes, and the number of issues (at least if it's truncated)